src/ci/docker/host-x86_64/armhf-gnu/vexpress_config

```diff
@@ -1932,7 +1932,7 @@ CONFIG_USB_ANNOUNCE_NEW_DEVICES=y
 CONFIG_USB_DEFAULT_PERSIST=y
 # CONFIG_USB_DYNAMIC_MINORS is not set
 # CONFIG_USB_OTG is not set
-# CONFIG_USB_OTG_WHITELIST is not set
+# CONFIG_USB_OTG_ALLOWLIST is not set
```

Comment:
?

Reply:
Yeah, this one doesn't make sense. Reverted.

Reply:
does _any_ of it make _any_ sense? 🤯